app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    name_file = command.split(" ")
+    print(command)
+    if name_file[1] != name_file[2] and name_file[0] == "cp":
+        with (open(f"{name_file[1]}", "r") as file_in,
+              open(f"{name_file[2]}", "w") as file_out):
```

Comment:
Using `with` statements for file operations is good practice, but consider adding error handling for cases where the files might not exist or cannot be opened. This will make the function more robust.